packages/roosterjs-editor-api/lib/format/toggleBullet.ts

```diff
@@ -1,9 +1,36 @@
 import execFormatWithUndo from './execFormatWithUndo';
-import { Editor } from 'roosterjs-editor-core';
+import getNodeAtCursor from '../cursor/getNodeAtCursor';
+import { Editor, browserData } from 'roosterjs-editor-core';
+import { NodeType } from 'roosterjs-editor-types';
+
+// Edge may incorrectly put cursor after toggle bullet, workaround it by adding a space.
+// The space will be removed by Edge after toggle bullet
+export function workaroundForEdge(editor: Editor): boolean {
+    if (browserData.isEdge) {
+        let node = getNodeAtCursor(editor);
+        if (node.nodeType == NodeType.Element && (<Element>node).textContent == '') {
+            (<Element>node).textContent = ' ';
+            return true;
+        }
+    }
+
+    return false;
+}
+
+export function removeWorkaroundForEdge(editor: Editor) {
+    let node = getNodeAtCursor(editor);
+    if (node.nodeType == NodeType.Element && (<Element>node).textContent == ' ') {
+        (<Element>node).textContent = '';
```

Comment:
It seems that you did nothing for the node.

Reply:
I changed its content from ' ' to ''.